charts/adaptive/values.yaml

```diff
@@ -63,6 +63,9 @@ harmony:
     tag: harmony:latest   # Add the harmony image tag
     pullPolicy: Always
 
+  group: default # Visible name of the group for this statefulset
+  partitionKey: default-statefulset # Partition name, should be unique across other groups
```

Comment:
@vdebergue it must be unique in its group, or globally unique?

Reply:
globally unique